src/crate/client/test_http.py

```diff
@@ -287,6 +290,18 @@ def test_socket_options_contain_keepalive(self):
         )
         client.close()
 
+    @patch(REQUEST, autospec=True)
+    def test_uuid_serialization(self, request):
+        client = Client(servers="localhost:4200")
+        request.return_value = fake_response(200)
+
+        uid = uuid.uuid4()
+        client.sql('insert into my_table (str_col) values (?)', (uid,))
```

Comment:
out of curiosity: is this trailing `,` of `(uid,)` required? (I see it in other tests as well.

Reply:
That turns it into a tuple in Python, as the client requires an iterable ;) 
```python
>>> type(("test"))
<class 'str'>
>>> type(("test",))
<class 'tuple'>

```

Reply:
Great, thx!